### PR TITLE
add automatic sections assignment when push to apple news

### PR DESCRIPTION
**Pull request by romulodl**
- create cron job to pull twice daily the sections from apple news API
 - create apple news settings page to set a default section
 - create custom field for categories to assign an apple news section
 - a bit of refactoring

### CODE REVIEW

File: app/Admin/AppleNewsSection.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+namespace AgreableAppleNewsPlugin\Admin;
+
+use AgreableAppleNewsPlugin\Services\Credentials;
+use AgreableAppleNewsPlugin\Services\API;
+
+class AppleNewsSection
+{
+	private $api = null;
+	private $taxonomy = 'apple_news_section';
+	private $event    = 'load_apple_news_secttions';
```
Comment (leonoragilmour) [inline app/Admin/AppleNewsSection.php]: spelling?

Comment (leonoragilmour) [reply app/Admin/AppleNewsSection.php]: should is be `load_apple_news_section`

Comment (romulodl) [reply app/Admin/AppleNewsSection.php]: nice catch! Fixed 👍

File: app/custom-fields/acf.php [hunk 2]
```diff
@@ -1,30 +1,114 @@
...
+				'type' => 'taxonomy',
+				'instructions' => '',
+				'required' => 0,
+				'conditional_logic' => 0,
+				'wrapper' => array (
+					'width' => '',
+					'class' => '',
+					'id' => '',
+				),
+				'taxonomy' => 'apple_news_section',
```
Comment (leonoragilmour) [inline app/custom-fields/acf.php]: This looks good - could potentially remove all the empty strings and default `0` ? just as a clean up?

Comment (romulodl) [reply app/custom-fields/acf.php]: 👍

**Pull request merged**